ipfs-cluster/trigger-server/README.md

```diff
@@ -1,3 +1,9 @@
+# What is trigger server
+
+The trigger server's purpose is defined in ipfs/ipfs-cluster/issues/244. The goal is to create an endpoint
```

Comment:
That issue is very long, and deals with other things. In any case, the user should not have to look up an issue in a different repository to find out what the project does, when reading's the project readme.